rvps/src/store/local_fs/mod.rs

```diff
@@ -13,7 +13,7 @@ use crate::{ReferenceValue, Store};
 
 /// Local directory path to store the reference values,
 /// which is created by sled engine.
-const FILE_PATH: &str = "/opt/attestation-server/reference_values";
+const FILE_PATH: &str = "/opt/confidential-containers/attestation-service/reference_values";
```

Comment:
Do we need to give a document to record all the paths that CoCo uses in all sub-projects? (e.g., AS, AA, image-rs, etc) I think it may help develop and test a lot

Reply:
Good idea.